cloudify_cli/commands/logs.py

```diff
@@ -83,6 +86,8 @@ def _download_archive(conn, host_type, output_path, logger):
 def download(output_path, all_nodes, logger):
     """Download an archive containing all of the manager's service logs
     """
+    output_json = {'archive paths': {}}
```

Comment:
A suggestion. Instead of passing this dictionary to the `_download_archive` function, make the `_download_archive` function return the `output_path` and use it in this function. 
I, personally, don't like modifying the same dictionary in different functions.